tagmanager/ctags/ctags.c

```diff
@@ -49,6 +49,7 @@
 
 
 
+#ifndef GLIB_MAJOR_VERSION
```

Comment:
IIUC `GLIB_MAJOR_VERSION` will always be defined, and if not, there will be a compilation (preprocessor) error much earlier where we unconditionally include `glib.h`.